flocker/acceptance/testtools.py

```diff
@@ -222,7 +225,12 @@ def _stop_acceptance_cluster():
     agent_nodes = filter(None, agent_nodes_env_var.split(':'))
 
     if control_node and agent_nodes:
```

Comment:
:question: Unrelated typo I noticed above when reading what this should do: accpetance > acceptance